src/qt/guiutil.cpp

```diff
@@ -666,10 +666,12 @@ QString ConnectionTypeToQString(ConnectionType conn_type, bool prepend_direction
 {
     QString prefix;
     if (prepend_direction) {
+        //: Connection direction. Also used in CONNECTION_TYPE_DOC and PeerTableModel::data().
```

Comment:
This translator comment will be assigned to the first string only, i.e., "Inbound".

We should add a translator comment to one string at a time (here and other places).

Reply:
Will address in a follow-up PR which focuses on improving these translations. For now, the commit referenced here has been dropped to make the PR focused on just adding a new `Direction` column. See: https://github.com/bitcoin-core/gui/pull/317#issuecomment-848432166